jekyll/_cci2/notifications.md

```diff
@@ -11,85 +11,84 @@ version:
 * TOC
 {:toc}
 
-
-CircleCI has integrated chat, email, and web notifications. Slack and Email notifications are delivered on the success or failure of a [workflow]({{ site.baseurl }}/2.0/workflows/). IRC notifications are delivered for each job. Consider the minimal CircleCI config below:
-
+CircleCI offers integrated email and web notifications. IRC and Slack notifications can also be configured using orbs. Slack, email, and web notifications are delivered on the succesful completion or failure of a [workflow]({{ site.baseurl }}/2.0/workflows/). IRC notifications are delivered for each job. Consider the minimal CircleCI config below:
 
 ```yaml
-version: 2
+version: 2.1
 jobs:
   build:
     docker:
       - image: cimg/<language>:<version TAG>
```

Comment:
I feel like this syntax is probably quite confusing, shall we just use the most basic image?

Reply:
Good idea!